packages/dx-react-grid/docs/reference/grid.md

```diff
@@ -1,6 +1,6 @@
 # Grid Reference
 
-The Grid is a root container component designed to process and display data specified via the `rows` property. You can configure columns using the `columns` property. The Grid's functionality (data visualization and data processing) is implemented in several plugins specified as child components. See the [plugins concept](../guides/plugin-overview.md) for details.
+The Grid is a root container component designed to process and display data specified via the `rows` property. You can configure columns using the `columns` property. The Grid's functionality (data visualization and data processing) is implemented in several plugins specified as child components. See the [plugins concept](../guides/plugin-overview.md) for details. Note that the Grid is a Plugin Host thus it can contain only plugins and plugin primitives.
```

Comment:
Please add a link to this article: https://devexpress.github.io/devextreme-reactive/react/core/docs/guides/fundamentals/#pluginhost-component

```suggestion
The Grid is a root container component designed to process and display data specified via the `rows` property. You can configure columns using the `columns` property.

The Grid's functionality (data visualization and data processing) is implemented in several [plugins](../guides/plugin-overview.md) specified as child components.

The Grid is a [PluginHost](), and as such, it can contain only other plugins and plugin primitives.
```